web/client/components/map/openlayers/__tests__/DrawSupport-test.jsx

```diff
@@ -13,6 +13,57 @@ const assign = require('object-assign');
 const DrawSupport = require('../DrawSupport');
 const {DEFAULT_ANNOTATIONS_STYLES} = require('../../../../utils/AnnotationsUtils');
 
+const viewOptions = {
+    projection: 'EPSG:3857',
+    center: [0, 0],
+    zoom: 5
+};
+const olMap = new ol.Map({
+    target: "map",
+    view: new ol.View(viewOptions)
+});
+const testHandlers = {
+    onStatusChange: () => {},
+    onGeometryChanged: () => {},
+    onEndDrawing: () => {},
+    onDrawingFeatures: () => {}
+};
+
+/* utility used to render the MeasurementSupport component with some default props*/
+const renderDrawSupport = (props = {}) => {
+    return ReactDOM.render(
+        <DrawSupport
+            features={props.features | []}
+            map={props.map || olMap}
+            onDrawingFeatures={testHandlers.onDrawingFeatures}
+            {...props}
+        />, document.getElementById("container"));
+};
+const renderThenClick = (props = {}) => {
+    let support = renderDrawSupport();
+
+    support = renderDrawSupport({
+        drawStatus: "drawOrEdit",
+        features: [props.feature],
+        options: {
+            drawEnabled: false,
+            editEnabled: true,
+            addClickCallback: true
+        },
+        ...props
+    });
+    support.props.map.dispatchEvent({
```

Comment:
i would not call it renderFeature because it does not render a feature. instead it renders the DrawSupport component that "does things" like drawing or enabling draw/modify interactions only.. and so on.
i'll remove the second dispatch.

i've created renderThenClick which includes the line "renderDrawSupport();" because it needs to enter will receive props and by doing this i'm saving it to being written every time i need renderThenClick.
eventually i can have two props one for the first on for the second.